src/sentry/replays/endpoints/project_replay_details.py

```diff
@@ -25,16 +36,42 @@ class ReplayDetailsPermission(ProjectPermission):
 
 
 @region_silo_endpoint
+@extend_schema(tags=["Replays"])
 class ProjectReplayDetailsEndpoint(ProjectEndpoint):
+    """
+    The same data as OrganizationReplayDetailsEndpoint, except project is required.
+    Query for a specific replay to fetch or delete.
+    """
+
     owner = ApiOwner.REPLAY
     publish_status = {
-        "DELETE": ApiPublishStatus.UNKNOWN,
-        "GET": ApiPublishStatus.UNKNOWN,
+        "DELETE": ApiPublishStatus.PUBLIC,
+        "GET": ApiPublishStatus.PUBLIC,
     }
 
     permission_classes = (ReplayDetailsPermission,)
 
+    @extend_schema(
+        operation_id="Retrieve a Replay Instance within a project",
+        parameters=[
+            GlobalParams.ORG_SLUG,
+            GlobalParams.PROJECT_SLUG,
+            ReplayParams.REPLAY_ID,
+            ReplayValidator,
```

Comment:
We only accept `start`, `end`, `stats-period`, and `field` on this endpoint.  You could abstract those fields out into a base class and then update the ReplayValidator to inherit from the base class.  The base class would then be the entry in this list.